### DISCUSSION TITLE
Is a `1.0.0` release on the horizon?

### DISCUSSION THREAD

**Discussion by dataders** [trinodb/trino-python-client#345] [General]
I think this would be a better user experience for devs downstream making apps the rely on this package. For example, having a three-part version would also enable compatibility pinning

**Comment by hashhar** [comment]
The client today intentionally doesn't guarantee compatibility and changing the versioning scheme itself won't make the client respect compatibility guarantees making version pinning not effective.

This is not planned for the near future.

**Comment by hovaesco** [comment]
Semantic versioning helps better manage such changes. With current versioning, we cannot do anything about it.
